extruct/xmldom.py

```diff
@@ -6,15 +6,22 @@
 from xml.dom import Node
 from xml.dom.minidom import Attr, NamedNodeMap
 
-from lxml.etree import (
-    ElementBase,
-    XPath,
-    _ElementStringResult,
-    _ElementUnicodeResult,
-    tostring,
-)
+from lxml.etree import ElementBase, XPath, _ElementUnicodeResult, tostring
 from lxml.html import HtmlElementClassLookup, HTMLParser
 
+try:
+    from lxml.etree import _ElementStringResult
+except ImportError:
+
+    class _ElementStringResult(bytes):  # type: ignore[no-redef]
+        """
+        _ElementStringResult is removed in lxml >= 5.1.0,
```

Comment:
```suggestion
        _ElementStringResult is removed in lxml >= 5.1.1,
```

Ref: https://github.com/lxml/lxml/commit/eba79343d0e7ad1ce40169f60460cdd4caa29eb3